docs/rendering/teisr.md

```diff
@@ -1,21 +1,21 @@
-TileEntityItemStackRenderer
+ItemStackTileEntityRenderer
 =======================
-!!! note
-    The features used here only exist in forge versions >= 14.23.2.2638.
-
-TileEntityItemStackRenderer is a method to use OpenGL to render on items.  This system is much simpler than the old TESRItemStack system, which required a TileEntity, and did not allow access to the ItemStack.
+ItemStackTileEntityRenderer is a method to use OpenGL to render on items. This system is much simpler than the old TESRItemStack system, which required a TileEntity, and did not allow access to the ItemStack.
 
-Using TileEntityItemStackRenderer
+Using ItemStackTileEntityRenderer
 --------------------------
 
-TileEntityItemStackRenderer allows you to render your item using `public void renderByItem(ItemStack itemStackIn)`.  
+!!! note
+    The ItemStackTileEntityRenderer is abbreviated as TEISR instead of ISTER as it used to be called TileEntityItemStackRenderer in previous versions.
```

Comment:
Just fix the abbreviation

Reply:
I kept it as TEISR for now because the method Item.Properties#setTEISR has the old abbreviation. I'll make a Forge PR to rename the method and already update the abbreviation here.